service/src/main/java/bio/terra/landingzone/library/configuration/MicrometerConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.landingzone.library.configuration;
+
+import io.micrometer.core.aop.CountedAspect;
+import io.micrometer.core.aop.TimedAspect;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
This is required only for @Counted/@Timed annotations. No usage now. Will remove it.